frontend/app/.server/domain/dtos/benefit-renewal.dto.ts

```diff
@@ -22,6 +22,16 @@ export type ItaChangeIndicators = Readonly<{
   hasAddressChanged: boolean;
 }>;
 
+export type ChildBenefitRenewalDto = BenefitRenewalDto & ChildChangeIndicators;
+
+export type ChildChangeIndicators = Readonly<{
+  hasAddressChanged: boolean;
+  hasEmailChanged: boolean;
+  hasMaritalStatusChanged: boolean;
+  hasPhoneChanged: boolean;
+  hasFederalProvincialTerritorialBenefitsChanged: boolean;
```

Comment:
It looks like the child flow doesn't explicitly ask whether the applicant's federal, provincial/territorial benefits have changed. If that's the case, we can remove `hasFederalProvincialTerritorialBenefitsChanged`.